### DISCUSSION TITLE
Tabs inaccessible with multiple windows per app on lxqt-panel - Fedora 34 in VM

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2101] [Q&A] [answered]
In addition to the other virtual machines (VM), I also created one running Fedora 34 LXQt Spin. I discovered when there are multiple windows open for an application, using Firefox and Thunderbird as examples, two windows each, if I click on the Thunderbird tab which displays 'Thunderbird - 2 windows', it displays the titles for both windows but when I mouse over to open one of them, they both disappear, essentially making Thunderbird inaccessible. The only way to get out of this is to right-click the tab and select Close Group, then re-launch Thunderbird.

This does not occur with the other VM's I'm using, which have Lubuntu and PCLinuxOS installed, nor does it occur with Debian 11 as the host OS. Does this sound like it may be an issue specific to Fedora, or perhaps because Fedora is being run in a VM? Fedora provides version 0.16 of LXQt.

I created an mp4 with my mobile showing what is occurring, but it's 24Mb in size (10 seconds) and I'm not sure if something like that can be attached to a message on GitHub.

_EDIT: In case it's a theme issue, it is using the (default) Fedora theme, but did not think to change the theme. at first. I am going to install the Silver and System themes from the GitHub package, as I've done with the other VM's._

_EDIT 2: Changed the theme to Silver, System and Light, they all do the same. So I will report this as a bug to Fedora._

_EDIT 3: Uploaded mp4 (10 seconds) to Google Drive, showing what it is doing:_ 

https://drive.google.com/file/d/1CbQoiYIGBl0scpdaMO4IQlSEhjNyVFpR/view?usp=sharing

Thanks in advance. 

P.S. I reported a separate bug (missing a main menu icon) to  Fedora and the assignee is _Orphan Owner_, so I suspect that perhaps Fedora does not have any LXQt maintainers.

**Comment by tsujan** [comment, marked as answer]
> but when I mouse over to open one of them, they both disappear,

It's a packaging problem in Fedora; some packages need to be rebuilt.

However, since LXQt 0.16 is outdated, rebuilding doesn't make much sense. Fedora needs an up-to-date version of LXQt. The latest release, as of this moment, is 0.17 but 1.0.0 will be released soon, in November.

**Reply by edwardp3** [reply]
Just out of curiosity, I downloaded the Fedora 35 Beta LXQt Spin, it has version 0.17 included. The problem described, has so far, not occurred with Fedora 35. I then installed it in another virtual machine and still no issues.
